Client/BookmarksHelper.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  BookmarksHelper.swift
+//  Client
+//
+//  Created by Nishant Patel on 6/10/21.
+//  Copyright © 2021 Mozilla. All rights reserved.
+//
+
+import Foundation
+import Storage
+
+class BookmarksHelper {
+
+    /// Filters bookmarks that are more than a specified number of days old.
+    /// - Parameters:
```

Comment:
Standard doc style is a blank line between discussion & params. :)
```suggestion
    ///
    /// - Parameters:
```